tests/README.md

```
@@ -1,105 +1,159 @@
-## Directory Structure
+## Test Organization: Where to Store Tests
 
-```bash
-tests/
-├── serve/              # E2E tests
-│   ├── conftest.py     # test fixtures as needed for specific test area
-├── conftest.py         # Shared fixtures and configuration
-├── utils               # Common utils accross tests
-└── README.md           # This file
+### Directory Structure
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)